EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -235,6 +244,48 @@ struct HfFilter { // Main struct for HF triggers
       LOG(fatal) << "Can't apply optimisation if ML is not applied.";
     }
   }
+
+  /// compute TPC postcalibrated nsigma based on calibration histograms from CCDB
+  /// \param objectMean calibration histograms of mean from CCDB
+  /// \param objectSigma calibration histograms of sigma from CCDB
+  /// \param track is the track
+  /// \param chargePar is the PID species
+  /// \return the corrected Nsigma value for the PID species
+  template <typename P, typename T>
+  double GetPostTPCCalib(const P& objectMean, const P& objectSigma, const T& track, const int chargePar)
```

Comment:
```suggestion
  double getTPCPostCalib(const P& objectMean, const P& objectSigma, const T& track, const int chargePar)
```
It is not clear from the names that `objectMean` and `objectSigma` are histograms.
Also what is the point of templating the histogram type if the function can digest only TH3F?